.github/workflows/javascript-test.yml

```diff
@@ -0,0 +1,19 @@
+name: Continuous testing
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use node 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
```

Comment:
Coming back to you with this one :D https://github.com/paritytech/opstooling-testing/pull/1/files#r1071349775

Reply:
Good catch. Time to update!